packages/shared/lib/auxiliary/deep-link/errors/amount-not-a-number.error.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseError } from '@core/error'
+
+export class AmountNotANumberError extends BaseError {
+    constructor(amountParam: string) {
+        const message = `Amount in deep link is not a number '${amountParam}'`
+        super({
+            message,
+            showNotification: true,
+            saveToErrorLog: true,
+            logToConsole: true,
```

Comment:
You can spread the `DEFAULT_APP_ERROR_PARAMETERS` constant here to save some code, although the only difference is that `logToConsole` is `false` since it's saved to the error log. It's also localized by default, so we should add a new locale entry for this one.

Reply:
I think it's more clear to have it explicit what happens here than to have one line of code less. Our error handling is not on point yet, so let's address this in a separate issue. I'll add the locale's however.